aci/resource_aci_aaaldapgroupmapruleref.go

```diff
@@ -45,7 +45,7 @@ func getRemoteLDAPGroupMapruleref(client *client.Client, dn string) (*models.LDA
 	}
 	aaaLdapGroupMapRuleRef := models.LDAPGroupMaprulerefFromContainer(aaaLdapGroupMapRuleRefCont)
 	if aaaLdapGroupMapRuleRef.DistinguishedName == "" {
-		return nil, fmt.Errorf("LDAPGroupMapruleref %s not found", aaaLdapGroupMapRuleRef.DistinguishedName)
+		return nil, fmt.Errorf("LDAP Group Map rule ref %s not found", dn)
```

Comment:
should we capitalise Rule Ref? In aci/resource_aci_aaaldapgroupmaprule.go you do use Rule capitalised.